src/rdb/rdb_util.c

```diff
@@ -218,6 +219,55 @@ rdb_anchor_from_hashes(struct rdb_anchor *anchor, daos_anchor_t *obj_anchor,
 	anchor->da_akey = *akey_anchor;
 }
 
+/*
+ * The nvops parameter must be equal to or larger than the number of VOS
+ * operations that will be executed in vtx.
+ */
+int
+rdb_vos_tx_begin(struct rdb *db, int nvops, rdb_vos_tx_t *vtx)
+{
+	struct dtx_handle *dth;
+#ifdef USE_VOS_TX
```

Comment:
[Note] This temporary compilation enables the current PR to build without the VOS changes. It will be removed once the VOS changes land.